### PR TITLE
Renamed 'input pressed' to 'float input changed' and raise only on changes to inputs

### PR DESCRIPTION
**Pull request by luis-valverde-ms**
This implements the changes suggested in issue #3722

### CODE REVIEW

**Comment by luis-valverde-ms** [comment]
This PR should also result in the of closing #3253.

File: Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityInputSystem.cs [hunk 1]
```diff
@@ -306,60 +306,45 @@ public interface IMixedRealityInputSystem : IMixedRealityEventSystem
...
+        #region Float Input Changed
 
         /// <summary>
-        /// Raise the input up event.
+        /// Raise Float Input Changed.
         /// </summary>
         /// <param name="source"></param>
         /// <param name="inputAction"></param>
-        void RaiseOnInputUp(IMixedRealityInputSource source, MixedRealityInputAction inputAction);
+        /// <param name="inputValue"></param>
```
Comment (wiwei) [inline Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityInputSystem.cs]: ><param name="inputValue"></param> [](start = 12, length = 33)

Can you leave some doccomments about the what this inputValue could be (i.e. is it between 0 and 1? Can it be negative? What does it represent?)

Or is it that it's so specific to the implementor that there's no reason to have this documented here?

Comment (luis-valverde-ms) [reply Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityInputSystem.cs]: The architecture is so generic that it is very difficult to make any assertions on what this value may represent. I think all we can say is: _this is a float value reported by an input source_.

Comment (wiwei) [reply Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityInputSystem.cs]: Sounds good!

**Pull request merged**